### ISSUE TITLE
no debugging symbols found

### ISSUE THREAD

**Issue by JorgeParalisis** [flatpak/flatpak#2719]
## Linux distribution and version
Ubuntu 18.04.2 LTS
## Flatpak version
1.0.7-0ubuntu0.18.04.1
## Description of the problem
I want to debug a gimp 2.10.8 crash according to wiki: Tips & Tricks from https://github.com/flatpak/flatpak/wiki/Tips-&-Tricks 
GIMP installation steps:
file downloaded = org.gimp.GIMP.flatpakref from https://flathub.org/apps/details/org.gimp.GIMP
installed via ubuntu software center
SDK installation via terminal: 
flatpak install flathub org.gnome.Sdk 3.28
also
flatpak install flathub org.gnome.Sdk//3.30
gbd reports: Reading symbols from gimp...(no debugging symbols found)...done.
## Steps to reproduce
open terminal
run:
flatpak run -d --command=sh org.gimp.GIMP
sh-4.3$ gdb gimp
GNU gdb (GDB) 7.11.1
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-unknown-linux".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from gimp...(no debugging symbols found)...done.
(gdb)

**Comment by mwleeds** [comment]
Try this command:
`flatpak install flathub org.gimp.GIMP.Debug`

If that works we should update the wiki

**Comment by JorgeParalisis** [comment]
Yeah, it works fine:
Reading symbols from gimp...Reading symbols from /usr/lib/debug//app/bin/gimp-2.10.debug...done.
done.
Thanks

**Comment by mwleeds** [comment]
I would just edit the wiki myself but I don't have a solid enough understanding of this stuff. [That page](https://github.com/flatpak/flatpak/wiki/Tips-&-Tricks) currently says you need to install the Sdk to get debug symbols, but aren't those in a separate extension for the runtime too, like `org.gnome.Platform.Debug`?

**Comment by JorgeParalisis** [comment]
yes, that was the problem, the Sdk doesn't install debug symbols (in my case gdb doesn't found) and an extra installation like org.gnome.Platform.Debug is needed, but i don't have precise information about why

**Comment by mwleeds** [comment]
You found that `org.gnome.Sdk` doesn't have debug symbols for the application (`org.gimp.GIMP`). Whether it has debug symbols for the runtime (`org.gnome.Platform`) is a separate question.

**Comment by JorgeParalisis** [comment]
OK thank you for the Clarification

**Comment by boredsquirrel** [comment]
https://docs.flatpak.org/en/latest/debugging.html
